content/2020/2020-07/2020-07-17-pulsar-weekly.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "2020-07-17-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2020-07-11 ~ 2020-07-17"
+date: 2020-07-11 ~ 2020-07-17
+description: "This is the Pulsar community weekly update for 2020-07-11 ~ 2020-07-17, with updates on Pulsar clients, broker, Functions, and so on."
+id: "2020-07-17-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-07-11 ~ 2020-07-17
+
+This is the Pulsar community weekly update for 2020-07-11 ~ 2020-07-17, with updates on Pulsar authentication, broker, Functions, and so on.
+
+### Notable Feature
+
+- [Pulsar Functions] Improve security setting of Pulsar Functions.
+
+    https://github.com/apache/pulsar/pull/7578
+
+- [Pulsar Functions] Support different docker images in Kubernetes runtime of Pulsar Functions.
+  
+    https://github.com/apache/pulsar/pull/6752
+
+- [Pulsar Client] Support Consumers Set Custom Retry Delay.
+  
+    https://github.com/apache/pulsar/pull/6449
+
+- [Build] Add limited module set profile to the POM files.
+  
+    https://github.com/apache/pulsar/pull/7541
+
+- [Build] Update Jersey to 2.31.
```

Comment:
```suggestion
- [Build] Update Jersey version to v2.31.
```

Reply:
same comments in  other similar cases.

Reply:
we need to refine the PR title if it's not clear/correct. You can check other PR titles in this file as well.